Gruntfile.js

```diff
@@ -243,7 +257,11 @@ module.exports = function (grunt) {
   });
 
   grunt.registerTask('default', ['newer:babel', 'less', 'newer:copy:dev', 'shell:electron', 'watchChokidar']);
-  grunt.registerTask('release', ['clean:release', 'babel', 'less', 'copy:dev', 'electron', 'copy:osx', 'shell:sign', 'shell:zip', 'copy:windows', 'rcedit:exes', 'compress']);
+  if(process.platform === 'linux') {
```

Comment:
@jeffdm while I'm looking at the code I'm wondering if we still need a `if` condition on the platform ?

I'm not familiar with Grunt but am I right to think that using the `electron` task, in the `release` task, will execute the `electron` task for each platform on the `grunt release` ?
In this case, merging both lines would still work and be cleaner, right ?